src/containers/Stories/index.jsx

```diff
@@ -12,10 +14,25 @@ const Stories = () => {
     const fetchPosts = async () => {
       const data = await fetch(_URL);
       const fetchedPosts = await data.json();
-      setPosts(fetchedPosts);
+      //filter posts based on the language
+      const filteredPosts = await fetchedPosts.filter((post) => {
+        return post.tags.toString() === langRef;
+      });
+      setPosts(filteredPosts);
     };
     fetchPosts();
-  }, []);
+  }, [langRef]);
```

Comment:
its better to do this for conditional rendering inorder not to cause alot of fetches whenever the language changes

Reply:
I did not understand hao should I change...
Can you give me more details?

Reply:
fixed